modules/sandboxed-containers-installing-operator-cli.adoc

```diff
@@ -3,5 +3,89 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-installing-operator-cli_{context}"]
+= Installing the {sandboxed-containers-operator} using the CLI
 
-= Installing the Sandboxed Containers Operator using the CLI
+You can install the {sandboxed-containers-operator} using the {product-title} CLI.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have subscribed to the {sandboxed-containers-first} catalog.
```

Comment:
Would it be possible to add a link to that procedure (I believe that you are referring to something like https://docs.openshift.com/container-platform/4.1/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli_olm-adding-operators-to-a-cluster)?